### DISCUSSION TITLE
Hoping to build a PikaReader box / future development

### DISCUSSION THREAD

**Discussion by lucai11** [PikaTimer/PikaReader#2] [General]
Just dropping a line to say I'm currently using PikaReader with some impinj readers. Used it as a backup in my last race and it worked great. I've built a little webapp to listen to the websocket and write to a txt file and rewind if required. 

Pretty keen to put together a real timing box with a battery, impinj and raspberry pi - any plans to get that little LCD status integration? I've got a java environment setup so could try and help.. but I'm not a java expert by any stretch.

I'm happy to share the webui/webapp with you if you'd like to see it as well - quite basic at this point

Thanks for developing this! was really tricky to find something that worked well with impinj and wasn't locked into certain timing software, this worked super smoothly and works with my r420 and r700

**Comment by segfaultcoredump** [comment]
The plan is to add LCD status display as well as GPIO support on the Raspberry PI platform for things like start/stop and trigger buttons as well as reading status lights. 

Take a look at the [TODO.md](https://github.com/PikaTimer/PikaReader/blob/master/TODO.md) file at the root of the directory for additional items.
